tests/conftest.py

```diff
@@ -118,6 +133,11 @@ async def async_http_request(
         {"sid": "uuid:dummy-avt1"},
         "",
     ),
+    ("UNSUBSCRIBE", "http://dlna_dmr:1234/upnp/event/QPlay"): (
+        200,
+        {"sid": "uuid:dummy-avt1"},
```

Comment:
When changing the sid for the SUBSCRIBE, remember to change this one too.

Reply:
Ok many thanks to both of you @StevenLooman @chishm.
I merged last edits in commits, so now commits are clean: one for client_factory edit and one for the tests.
Pre-commit tests and pytest ran locally get passed, build on GitHub side is still marked as failed.
Let me know if we are on the right way :)